bigchaindb/__init__.py

```diff
@@ -24,6 +23,9 @@ def e(key, default=None, conv=None):
 
 
 config = {
+    'server': {
+        'bind': e('BIGCHAIN_SERVER_BIND', default='0.0.0.0:5000'),
```

Comment:
should we allow it to bind to all interfaces by default or should we force the user the change the default?